src/lib/blueprint.ts

```diff
@@ -1,7 +1,33 @@
 import type { Openapi } from './openapi.js'
 
+interface Parameter {
+  name: string
+  type: string
+  required: boolean
+  description: string
+}
+
+interface ReturnValue {
+  type: string
+  description: string
+}
+
+interface Example {
+  codeSnippet: string
+  explanation: string
+}
+
 export interface Blueprint {
   name: string
+  routes: {
+    path: string
+    method: string
+    methodDescription: string
```

Comment:
What is this methodDescription?

Reply:
<img width="951" alt="Screenshot 2024-06-21 at 00 15 23" src="https://github.com/seamapi/blueprint/assets/41388251/46d29331-a6c0-4fd6-a147-9e2064c60b1a">
That's what I had in mind

Reply:
How is this different than just the route description?

Reply:
It's not. I just couldn't find a good name :)